pages/api/auth/[...nextauth].js

```diff
@@ -1,41 +1,62 @@
-import NextAuth from 'next-auth'
+import NextAuth from '@dts-stn/next-auth'
 
 // For more information on each option (and a full list of options) go to
 // https://next-auth.js.org/configuration/options
 export default NextAuth({
   // https://next-auth.js.org/configuration/providers/oauth
   providers: [
-    /* EmailProvider({
-         server: process.env.EMAIL_SERVER,
-         from: process.env.EMAIL_FROM,
-       }),
-    */
     {
-      id: 'customProvider',
-      name: 'customProvider',
+      id: 'ecasProvider',
+      name: 'ECAS',
       clientId: process.env.CLIENT_ID,
-      clientSecret: process.env.CLIENT_SECRET,
       type: 'oauth',
-      wellKnown: process.env.WELL_KNOWN,
```

Comment:
If we are removing key cloak, what do you think about removing the env variables for it as well (docker/team city files)? Maybe at least give them less generic names.